yahoofantasy/__init__.py

```diff
@@ -7,4 +7,5 @@
 from .resources.week import Week  # noqa: F401
 from .stats.stat import Stat  # noqa: F401
 
-__version__ = "1.3.1"
+__version__ = "1.3.2"
```

Comment:
You can remove this version bump from this PR. I will run the version upgrade manually after it's merged. It would be a minor release (1.4.0) vs a patch release (1.3.2) anyway, since this is all new functionality

Reply:
made these changes, also made some more in the README now that all of the sports are supported 🥳